src/installer.js

```diff
@@ -91,6 +91,24 @@ var readLicense = function (options, callback) {
 }
 
 /**
+ * Determine the homepage based on the settings in `package.json`.
+ */
+var getHomepage = function (pkg) {
+  if (pkg.homepage) {
+    return pkg.homepage
+  } else if (pkg.author) {
+    if (typeof pkg.author === 'string') {
+      var homepageRegexp = /.*\(([^)]+)\).*/
```

Comment:
Not a big deal, but FWIW, this could also be accomplished by using [`parse-author`](https://github.com/jonschlinkert/parse-author), which might help avoid having to duplicate parsing/regex functionality here.